notflix

```diff
@@ -2,71 +2,71 @@
 
 # Dependencies - webtorrent, mpv
 
-mkdir -p $HOME/.cache/notflix
+mkdir -p "$HOME"/.cache/notflix
 
 menu="dmenu -i -l 25"
 baseurl="https://1337x.wtf"
 cachedir="$HOME/.cache/notflix"
 
-if [ -z $1 ]; then
+if [ -z "$1" ]; then
   query=$(dmenu -p "Search Torrent: " <&-)
 else
   query=$1
 fi
 
-query="$(echo $query | sed 's/ /+/g')"
+query="$(echo "$query" | sed 's/ /+/g')"
```

Comment:
It is best not to put quotes around this query variable. Without quotes, if the user entered something like "     Suicide Squad     " with a bunch of leading and trailing whitespaces, echoing it would automatically remove the leading and trailing spaces and only the space between the words would get replaced by plus sign in the following sed command. So, the output query would be "Suicide+Squad".

Whereas, if you put quotes around the query variable while echoing, it wouldn't remove the extra spaces and all those extra spaces would get converted to plus signs in the following sed command. In this case the output query would be something like "++++++Suicide+Squad++++"